src/lib/swapper/swappers/ThorchainSwapper/utils/addAggregatorAndDestinationToMemo.ts

```diff
@@ -36,6 +38,20 @@ export const addAggregatorAndDestinationToMemo = ({
     'expected finalAssetLimitWithManualSlippage to be a positive amount',
   )
 
+  const minimumPrecision = 5
```

Comment:
Not so much of a minimum if we do `finalAssetPrecision - minimumPrecision` below

Reply:
Also we should really never have assets with a precision of less than 5 - since this heuristic was chosen arbitrarily, can we perhaps do 6 and have the checks below *inclusive*? i.e USDC and USDT both have 6 dp